adafruit_display_notification/apple.py

```diff
@@ -13,13 +13,22 @@
 
 from . import PlainNotification
 
+try:
+    from adafruit_ble_apple_notification_center import Notification
```

Comment:
I think we'll want to have an "unused" typing import in this try block like this:

```
try:
    from typing import List  # pylint: disable=unused-import
    from adafruit_ble_apple_notification_center import Notification
except ImportError:
    pass
```
The purpose of that typing import is to cause the ImportError to be raised (and then ignored) when this code runs on a microcontroller which means that the `Notification` import won't occur and will save some RAM since it wouldn't actually be used at runtime.

Unfortunately pylint doesn't know this context so it believes the import is unused, therefore we also need a directive to it to let it allow this import even though it's not used in the sense that it is meaning.

In most files we don't have to have an "unused" import like this because typically there will be some actually typing import that is needed and used within the file. In this case the only import is this one which isn't part of typing even though that is all it's needed for.

Reply:
Yes, I understood you point. I also added a comment to clarify the use of the unused import. This should be removed as the file grows and other typing imports are added in future.
Thanks for your review!

Reply:
Could you (@FoamyGuy) please review the changes again or should I open a new PR?